src/components/CoverageStatementsMonograph/CoverageStatementsMonograph.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+import { Layout } from '@folio/stripes/components';
+
+export default class CoverageStatementsMonograph extends React.Component {
+  static propTypes = {
+    pci: PropTypes.shape({
+      pti: PropTypes.shape({
+        titleInstance: PropTypes.shape({
+          dateMonographPublished: PropTypes.string,
+          monographVolume: PropTypes.string,
+          monographEdition: PropTypes.string,
+          name: PropTypes.string
+        }),
+      }).isRequired
+    }),
+  }
+
+  renderEdition(edition) {
+    if (!edition) return null;
+
+    return (
+      <React.Fragment>
+        <FormattedMessage id="ui-agreements.coverage.editionShort" />
+        {edition}
+      </React.Fragment>
+    );
+  }
+
+  renderVolume(volume) {
+    if (!volume) return null;
+
+    return (
+      <React.Fragment>
+        <FormattedMessage id="ui-agreements.coverage.volumeShort" />
+        {volume}
+      </React.Fragment>
+    );
+  }
+
+  renderCoverage(pci) {
+    // Date can take the forms yyyy, yyyy-mm or yyyy-mm-dd, and is stored as a string.
+    const date = get(pci, 'pti.titleInstance.dateMonographPublished');
+    const volume = get(pci, 'pti.titleInstance.monographVolume');
+    const edition = get(pci, 'pti.titleInstance.monographEdition');
+
+    if (!date && !volume && !edition) {
+      return '*';
+    } else {
```

Comment:
Note that when you return out of an `if` like you do on L50, you don't need this `else` statement and can break the contents of it back out into the parent scope.